core-services/image-mirroring/openshift/mapping_origin_4_12

```diff
@@ -145,6 +145,7 @@ registry.ci.openshift.org/origin/4.12:gcp-pd-csi-driver quay.io/openshift/origin
 registry.ci.openshift.org/origin/4.12:gcp-pd-csi-driver-operator quay.io/openshift/origin-gcp-pd-csi-driver-operator:4.12 quay.io/openshift/origin-gcp-pd-csi-driver-operator:4.12.0 quay.io/openshift/origin-gcp-pd-csi-driver-operator:latest
 registry.ci.openshift.org/origin/4.12:gcp-pd-csi-driver-operator-test quay.io/openshift/origin-gcp-pd-csi-driver-operator-test:4.12 quay.io/openshift/origin-gcp-pd-csi-driver-operator-test:4.12.0 quay.io/openshift/origin-gcp-pd-csi-driver-operator-test:latest
 registry.ci.openshift.org/origin/4.12:ghostunnel quay.io/openshift/origin-ghostunnel:4.12 quay.io/openshift/origin-ghostunnel:4.12.0 quay.io/openshift/origin-ghostunnel:latest
+registry.ci.openshift.org/origin/4.12:git-client quay.io/openshift/origin-git-client:4.12 quay.io/openshift/origin-git-client:4.12.0 quay.io/openshift/origin-git-client:latest
```

Comment:
Please remove that and add that image in your promotion config as `excluded_images:`

Reply:
Done.Thanks for your for help.